crates/subspace-verification/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "subspace-verification"
+version = "0.1.0"
+authors = ["Vedhavyas Singareddi <ved@subspace.network>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://subspace.network"
+repository = "https://github.com/subspace/subspace"
+description = "Verification primitives for Subspace"
+readme = "README.md"
```

Comment:
I know we're not very consistent with this, but I'd recommend including `include` section and list publishable files explicitly